woocommerce/woocommerce-gutenberg-products-block
# Shopper → Checkout → Can choose payment option

Fixes #5782

### Notes

This PR adds e2e tests to verify that a shopper can choose the default payment options (`Direct bank transfer`, `Cash on delivery` and `Check payments`) on the checkout block. 

### Testing

#### Automated Tests

### Manual Testing

1. Check out this PR.
2. Start Docker Desktop.
3. Run `npm run wp-env destroy && npm run wp-env start` to start a fresh e2e environment.
4. Run `npm run test:e2e -- tests/e2e/specs/shopper/checkout-payments.test.js` to run only this test case.
5. Run `npm run test:e2e -- tests/e2e/specs/shopper/*` to run all shopper-related test cases.
6. Run `npm run test:e2e` to run the whole test suite.
